Sources/StevensonCore/CommandHandler.swift

```diff
@@ -0,0 +1,18 @@
+import Vapor
+
+public struct Command {
+    public let name: String
+    public let arguments: [String: String]
+    public let branch: String?
```

Comment:
A little confusing that both Slack commands to be parsed and CI commands to be triggered on CircleCI both have the name `Command`. iiuc, this `public struct Command` is to describe a CircleCI command to be triggered, right? So might be worth either namespacing it or rename it `CICommand`?

Reply:
slack command for this bot is just a name and json payload that is being received through the endpoint. It's being transformed by the bot then to the format that CircleCI can understand. We can treat it with CQS defining model that we parse from the json as query and transforming it to command, but not sure what practical benefit it will have.